data_loader/test/test_training_interface_logic.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from data_loader.training_storer import get_values, insert_values
+
+TEST_INSERT_SPREADSHEET_ID = '1KQ2nFO9oeDLhHmKh9UTvNnhepKhMuTOyKmh7Nnxbj80'
+
+
+class TestTrainingInterfaceLogic(unittest.TestCase):
+    def test_get_values_data(self):
+        values = get_values()
+        assert all(isinstance(row, list) for row in values), 'Function returning invalid data'
+
+    def test_get_values_not_empty(self):
+        values = get_values()
+        assert len(values) != 0, 'Function returned an empty list'
+
+    def test_insert_values(self):
+        cells = [['a', 'b', 'c']]
+
+        response = insert_values(cells, TEST_INSERT_SPREADSHEET_ID)
+
+        assert response, 'Request returned bad response'
```

Comment:
good start here usually when you test dao you want to:
1- insert, verify what you inserted was...inserted.
2- delete what you inserted, verify it was deleted.

what I see is that you mostly check no exception was raised with some  very basic sanity testing.